### ISSUE TITLE
Add other twilights

### ISSUE THREAD

**Issue by jareyes** [darkskyapp/astro#7]
I believe we use civil twilight to calcultae dawn and dusk (but I don't remember and am too lazy to check). We might as well include all three: civil, nautical, and astronomical twilight.

**Comment by ironwallaby** [comment]
Yes, `dawn` and `dusk` currently refer to civil twilight in the API.
